src/nextjs-front/pages/friends.tsx

```diff
@@ -107,7 +107,8 @@ const FriendsPage: NextPageWithLayout = ({}) => {
                   content="Edit profile"
                 >
                   <button className="p-2 text-2xl text-white/90 bg-01dp rounded-full transition hover:scale-105">
-                    <RiUserSettingsLine
+                    <CogIcon
```

Comment:
A button that use the router on click could simply be implemented as a link.

Reply:
NextJS links have several pros.
For one, NextJS actually preloads the page that are pointed by links on the currently viewed page.